x/claim/client/cli/tx.go

```diff
@@ -20,7 +20,10 @@ func GetTxCmd() *cobra.Command {
 		RunE:                       client.ValidateCmd,
 	}
 
-	cmd.AddCommand(CmdClaimInitial())
+	cmd.AddCommand(
+		CmdClaimInitial(),
+		CmdClaim(),
```

Comment:
```suggestion
		CmdClaim(),
		CmdClaimInitial(),
```
I think `MsgClaim` is the main most important message of the module